client/src/components/ReportCollection.js

```diff
@@ -47,39 +52,63 @@ const ReportCollection = ({
         {showHeader && (
           <header>
             {viewFormats.length > 1 && (
-              <ButtonToggleGroup
-                value={viewFormat}
-                onChange={setViewFormat}
-                className="d-print-none"
-              >
-                {viewFormats.includes(FORMAT_TABLE) && (
-                  <Button value={FORMAT_TABLE} variant="outline-secondary">
-                    Table
-                  </Button>
-                )}
-                {viewFormats.includes(FORMAT_SUMMARY) && (
-                  <Button value={FORMAT_SUMMARY} variant="outline-secondary">
-                    Summary
-                  </Button>
-                )}
-                {viewFormats.includes(FORMAT_CALENDAR) && (
-                  <Button value={FORMAT_CALENDAR} variant="outline-secondary">
-                    Calendar
-                  </Button>
+              <>
+                <ButtonToggleGroup
+                  value={viewFormat}
+                  onChange={setViewFormat}
+                  className="d-print-none"
+                >
+                  {viewFormats.includes(FORMAT_TABLE) && (
+                    <Button value={FORMAT_TABLE} variant="outline-secondary">
+                      Table
+                    </Button>
+                  )}
+                  {viewFormats.includes(FORMAT_SUMMARY) && (
+                    <Button value={FORMAT_SUMMARY} variant="outline-secondary">
+                      Summary
+                    </Button>
+                  )}
+                  {viewFormats.includes(FORMAT_CALENDAR) && (
+                    <Button value={FORMAT_CALENDAR} variant="outline-secondary">
+                      Calendar
+                    </Button>
+                  )}
+                  {viewFormats.includes(FORMAT_MAP) && (
+                    <Button value={FORMAT_MAP} variant="outline-secondary">
+                      Map
+                    </Button>
+                  )}
+                  {viewFormats.includes(FORMAT_STATISTICS) && (
+                    <Button
+                      value={FORMAT_STATISTICS}
+                      variant="outline-secondary"
+                    >
+                      Statistics
+                    </Button>
+                  )}
+                </ButtonToggleGroup>
+                {viewFormat === FORMAT_CALENDAR && (
+                  <ButtonToggleGroup
+                    value={calendarAttendeeType}
+                    onChange={setCalendarAttendeeType}
+                    className="float-md-end"
+                  >
+                    <Button
+                      value={Settings.fields.advisor.person.name}
```

Comment:
I would distinguish between the *value* of the button and the *label*. For the button values I would use hard-coded `advisor` and `interlocutor`, for the labels I would use the dictionary settings like you do already.